data/json/items/armor.json

```
@@ -6649,7 +6649,7 @@
     "name": "protective mouthpiece",
     "description": "A protective piece of equipment commonly used by athletes, which is worn inside the mouth to protect your teeth.",
     "weight": 20,
-    "volume": 0,
+    "volume": "0 L",
```

```suggestion
    "volume": "430 ml",
```
Going off of [this mouthpiece](https://www.amazon.com/Shock-Doctor-Mouthguard-Football-Basketball/dp/B07BZDTQ4R) of these dimensions 9.5 x 7.5 x 2 inches, then cutting it in half.